stock_request/models/stock_move_line.py

```diff
@@ -10,27 +12,34 @@ class StockMoveLine(models.Model):
     @api.model
     def _stock_request_confirm_done_message_content(self, message_data):
         title = (
-            _("Receipt confirmation %(picking_name)s for your Request %(request_name)s")
+            Markup("<h3>%s<h3>")
+            % _(
+                "Receipt confirmation %(picking_name)s for your Request "
+                "%(request_name)s"
+            )
```

Comment:
@ced-adhoc You should use translation function arguments to pass translation value. `_("%(request_name)s", message_data)`